wbb/modules/music.py

```diff
@@ -39,9 +45,10 @@ async def music(client, message):
         duration = int(float(info_dict['duration']))
         caption = f"<b><a href=\"{webpage_url}\">{title}</a></b>"
         await message.reply_chat_action("upload_document")
-        await message.reply_audio(audio_file, caption=caption,
-                                  duration=duration, performer=performer,
-                                  title=title, thumb=thumbnail_file)
+        await message.reply_audio(
+            audio_file, caption=caption, duration=duration,
+            performer=performer, title=title, thumb=thumbnail_file
+            )
         os.remove(audio_file)
```

Comment:
You shouldn't change this, the way you write is a bit confusing.
each parameter or the way I wrote before are both fine.

the way you wrote should be avoid and only use when there are parameters which is very long.

Reply:
btw I think ) takes one line looks a bit strange, maybe that's also uncommon.

Reply:
i only edited it because flake8 was giving indentation erorrs

Reply:
what indentation errors? I don't think it gives errors on that. if it throw errors on that, we may already notice that in CI log

Reply:
you can check the CI log of dev branch, it does not throw errors about lint.

Reply:
undoing those changes